### DISCUSSION TITLE
Show other formats in HTML

### DISCUSSION THREAD

**Discussion by jdc-pub** [quarto-dev/quarto-cli#3613] [Feature Requests]
For websites, blogs, and HTML books, it would be super cool if there was a feature that noticed if you have other rendered formats (`keep-ipynb`, `keep-md`, etc), and automatically added "Download as a Jupyter Notebook" and/or "Download Markdown" to the top bar, or side bar, or something.

**Comment by cscheid** [comment]
That already exists in the `v1.3` prerelease versions. The issue you're running into is that `keep-ipynb` and `keep-md` are not meant to be "othered rendered formats": they're meant to be used as debugging aids in case your rendering goes wrong. Do this instead:

```
---
title: Multiple Formats
format:
  html: default
  ipynb: default
  markdown+gfm: default
---

This is a test of multiple formats.
```

You'll get this rendering:

<img width="1032" alt="image" src="https://user-images.githubusercontent.com/285675/206726317-b96c140c-ac00-440d-9a9f-20b048350cd6.png">

**Comment by mcanouil** [comment]
Closing as completed as part of Quarto 1.3, see <https://quarto.org/docs/output-formats/html-multi-format.html#specifying-formats-to-link>.
